recipes/mimalloc/config.yml

```
@@ -0,0 +1,3 @@
+versions:
+  "v1.6.4":
```

That's why the CI is failing.
```suggestion
  "1.6.4":
```